examples/undocumented/libshogun/streaming_vwfeatures.cpp

```diff
@@ -55,12 +58,10 @@ int main()
 		SG_SPRINT("For example %d, predicted label is %f.\n", i, test_labels->get_label(i));
 
 	SG_UNREF(test_features);
-	SG_UNREF(test_file);
-	SG_UNREF(train_features);
-	SG_UNREF(train_file);
```

Comment:
this issue is fixed due to the bug at src/shogun/features/streaming/StreamingStringFeatures.h